scipy/stats/tests/test_discrete_distns.py

```diff
@@ -99,6 +100,81 @@ def test_betabinom_bernoulli():
     expected = bernoulli(a / (a + b)).pmf(k)
     assert_almost_equal(p, expected)
 
+
+def test_issue_10317():
+    alpha, n, p = 0.9, 10, 1
+    assert_equal(nbinom.interval(alpha=alpha, n=n, p=p), (0, 0))
+
+
+def test_issue_11134():
+    alpha, n, p = 0.95, 10, 0
+    assert_equal(binom.interval(alpha=alpha, n=n, p=p), (0, 0))
+
+
+def test_issue_7406():
+    np.random.seed(0)
+    assert_equal(binom.ppf(np.random.rand(10), 0, 0.5), 0)
+
+
+def test_binom_ppf_endpoints():
```

Comment:
Did this correspond with an issue?

Reply:
I don't think so, but I think you mentioned wanting to check the endpoints.  I'd have to dig through all our past messages to find it

Reply:
Oh. Hah oops.

Reply:
Looks like this test originated [here](https://github.com/scipy/scipy/issues/7406#issuecomment-743720925):
> Also, I'm curious what it does for `q = 0` and `q = 1`. Another one for the list!

Happy to remove it if this test is not doing anything useful.